enumeration/alerter/alert.go

```diff
@@ -22,6 +27,10 @@ func (f *FakeAlert) ShouldIgnoreResource() bool {
 	return f.IgnoreResource
 }
 
+func (f *FakeAlert) Resource() *resource.Resource {
```

Comment:
Is this FakeAlert still used anywhere?

Reply:
Yeah in some tests, it's a manual mock if you see what I mean.

Reply:
Shouldn't we move it into the test files then?

Reply:
Yeah agreed, that's typically tech debt here.

Reply:
you're right, it's out of scope for this PR! I forgot I was commenting on the new interface method, not a whole new struct!